operations/pyroscope/helm/pyroscope/templates/deployments-statefulsets.yaml

```diff
@@ -81,6 +81,13 @@ spec:
               {{- end }}
           {{- end }}
           {{- end }}
+          env:
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: NAMESPACE_FQDN
+              value: "{{ .Release.Namespace }}.svc{{ .Values.pyroscope.cluster_domain }}"
```

Comment:
I'm not fond of doing this, but I couldn't find a simpler solution. I don't want to involve DNS here